model/flow/identifier.go

```diff
@@ -64,6 +66,16 @@ func (id Identifier) Format(state fmt.State, verb rune) {
 	}
 }
 
+// IsSampled is a utility method to sample entities based on their ids,
+// percentage adjust how often we sample the entity.
+// 0 means 0 data will be collected.
+// 50 means 50% of the entities would be collected.
+// 100 means all data will be collected.
+func (id Identifier) IsSampled(percentage uint) bool {
```

Comment:
can we add a test case for this function?

Reply:
I think using float with range 0..1.0 is more common